src/op_analytics/cli/subcommands/pulls/app.py

```diff
@@ -4,7 +4,10 @@
 from op_coreutils.logger import structlog
 
 from .agora import agora_pull
-from .defillama import pull_stablecoins as dfl_pull_stablecoins
+from .defillama.defillama_stablecoins import pull_stablecoins as dfl_pull_stablecoins
```

Comment:
the import has a redundant ring to it.  It might be better if it was `from .defillama.stablecoins`  and `from .defillama.historical_chain_tvl` 

Also on the `historical` qualifier.  I suggest we drop it from the name.  Something like 
```
from .defillama.tvl_chains
```
I wasn't sure what the `historical` name was implying.  

Also the reason I suggest `tvl_chains` is that then the modules would be sorted alphabetically more nicely if you later on add something like `tvl_apps`.

Reply:
it's a reference to the name of the end point. Defillama has chains, tvl, and historicalchainstvl end points. I wanted to keep it as similar to the end point name as possible so it's super clear what exactly we're pulling

open to changing though and decoupling from the naming conventions within defillama

Reply:
ah that makes perfect sense then let's honor the API names